RUFAS/biophysical/animal/ration/ration_optimizer.py

```diff
@@ -590,8 +618,8 @@ def fat_constraint(decision_vector: npt.NDArray[np.float64], ration_configuratio
         ----------
         decision_vector : numpy.ndarray
             The decision vector used in the scipy minimize method.
-        ration_config: RationConfig object
-            Attributes are animal requirement and feed supply information required for optimization
+        ration_configuration: RationConfig object
+            Attributes are animal requirement and feed supply information required for optimization.
```

Comment:
```suggestion
        ration_configuration: RationConfig
            Collection of animal requirements and feed supply information for the ration formulation process.
```
The current definition is a little confusing to me so I just subbed in the class definition from the top of the module.